src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/TasksController.kt

```diff
@@ -89,9 +89,53 @@ class TasksController(
     val tasks = mutableListOf<Task>()
 
     if (user.hasRole(UserRole.CAS1_MATCHER)) {
-      async { tasks += getPlacementRequestTasks(placementRequestService.getVisiblePlacementRequestsForUser(user), user) }
+      async {
+        tasks += getPlacementRequestTasks(
+          placementRequestService.getVisiblePlacementRequestsForUser(user),
+          user,
+        )
+      }
+
+      async {
+        tasks += getPlacementApplicationTasks(
+          placementApplicationService.getVisiblePlacementApplicationsForUser(user),
+          user,
+        )
+      }
+    }
+
+    return@runBlocking ResponseEntity.ok(tasks)
+  }
+
+  override fun tasksTaskTypeGet(taskType: String): ResponseEntity<List<Task>> = runBlocking {
+    val user = userService.getUserForRequest()
+    var tasks = listOf<Task>()
```

Comment:
I think this will just return an empty list now?

Probs better to do:

```suggestion
var tasks = if (user.hasRole(UserRole.CAS1_MATCHER)) {
      when (type) {
        TaskType.placementApplication -> {
          async {
            tasks = getPlacementApplicationTasks(
              placementApplicationService.getVisiblePlacementApplicationsForUser(user),
              user,
            )
          }
        }
        TaskType.placementRequest -> {
          async {
            tasks = getPlacementRequestTasks(
              placementRequestService.getVisiblePlacementRequestsForUser(user),
              user,
            )
          }
        }
        else -> {
          throw BadRequestProblem()
        }
      }
```